PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -154,15 +212,27 @@ struct HfCandidateSelectorLc {
     // looping over 3-prong candidates
     for (const auto& candidate : candidates) {
 
-      // final selection flag: 0 - rejected, 1 - accepted
+      // final selection flag
       auto statusLcToPKPi = 0;
       auto statusLcToPiKP = 0;
 
+      auto ptCand = candidate.pt();
+
```

Comment:
Here you should also clear the content of `outputMlLcToPKPi` and `outputMlLcToPiKP`. Otherwise, a wrong output will be filled in the table for candidates who do not reach the ML selection part. See https://github.com/AliceO2Group/O2Physics/blob/master/PWGHF/TableProducer/candidateSelectorDsToKKPi.cxx#L218-L223 for an example.